micro-couchbase/src/main/java/com/aol/micro/server/couchbase/CouchbaseDistributedMapClient.java

```diff
@@ -3,10 +3,10 @@
 import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 
+import com.aol.cyclops2.util.ExceptionSoftener;
```

Comment:
Why is it still com.aol.cyclops(2) here while it's cyclops.X elsewhere. Is this API not intended for use outside cyclops?